rg_instructor_analytics/static/rg_instructor_analytics/css/instructor_analytics.css

```diff
@@ -696,6 +697,10 @@
     box-sizing: border-box;
     padding: 0 3px;
 }
+#problems-stats-plot .plot-bar-vertical {
```

Comment:
why do we need this change in addition to the L534?

Reply:
there is two different blocks named ```plot-bar-vertical``` and ```plot-bar-vert``` with different rules so in this case we have 2 additional lines of code , if i make new rule for both classes it will be 3 lines , and harder to find

Reply:
got it, thanks